opensearch-operator/pkg/builders/cluster.go

```diff
@@ -457,7 +462,8 @@ func NewSTSForNodePool(
 			Selector: &metav1.LabelSelector{
 				MatchLabels: matchLabels,
 			},
-			PodManagementPolicy: appsv1.OrderedReadyPodManagement,
+			PodManagementPolicy:                  appsv1.OrderedReadyPodManagement,
+			PersistentVolumeClaimRetentionPolicy: persistentVolumeClaimRetentionPolicy,
```

Comment:
Thanks, I'm curious what would be the default when `persistentVolumeClaimRetentionPolicy` is not passed? Can we please handle that as well.